bin/k8s-conjur-deploy/bootstrap.env

```diff
@@ -0,0 +1,52 @@
+export CONJUR_VERSION=5
+export CONJUR_APPLIANCE_IMAGE=registry.tld/conjur-appliance:5.0-stable
+export CONJUR_NAMESPACE=[namespace to deploy Conjur to]
+export CONJUR_NAMESPACE_NAME=[namespace to deploy Conjur to]
+export AUTHENTICATOR_ID=[unique authenticator ID for this deployment]
+export CONJUR_FOLLOWER_COUNT=1
+export PLATFORM=openshift
+export OPENSHIFT_USERNAME=cadmin
+export OPENSHIFT_PASSWORD=Mdm2n7MCMAmzgp2rKft
+export OSHIFT_CLUSTER_ADMIN_USERNAME=cadmin
+export OSHIFT_CONJUR_ADMIN_USERNAME=cadmin
+export DOCKER_REGISTRY_PATH=docker-registry-default.openshift-311.itci.conjur.net
+export DEPLOY_MASTER_CLUSTER=true
+export CONJUR_ACCOUNT=[name of conjur account with which to authenticate to seed service]
+export CONJUR_ADMIN_PASSWORD=[conjur admin password]
+
```

Comment:
can you please have all the variables that need to be updated by the user in one place?